Library/src/main/java/dev/testify/output/DataDirectoryDestination.kt

```diff
@@ -0,0 +1,75 @@
+package dev.testify.output
+
+import android.content.Context
+import android.graphics.Bitmap
+import android.graphics.BitmapFactory
+import android.util.Log
+import dev.testify.internal.DEFAULT_FOLDER_FORMAT
+import dev.testify.internal.DeviceStringFormatter
+import dev.testify.internal.formatDeviceString
+import java.io.File
+import java.io.FileOutputStream
+
+open class DataDirectoryDestination(
+    context: Context,
+    fileName: String,
+    extension: String = PNG_EXTENSION,
+    val key: String? = null,
+    val root: String? = null
+) : Destination {
+
+    companion object {
+        private const val DATA_DESTINATION_DIR = "images"
+        private const val LOG_TAG = "DataDirectory"
```

Comment:
in the `SdCardDestination` this tag named `SdCardDestination` - maybe worth naming it here in the uniform way? 🤔

Reply:
I would have named this `DataDirectoryDestination` but there's a lint warning for log tags over 23 characters, so I truncated it